tests/js/spec/components/__snapshots__/issueSyncListElement.spec.jsx.snap

```diff
@@ -6,6 +6,10 @@ exports[`AlertLink renders 1`] = `
     className="css-17m6soy-hoverCardStyles"
     containerClassName="css-1s8b4pt-hoverCardContainer"
     displayTimeout={100}
+    dynamicWidth={false}
+    maxWidth={295}
```

Comment:
passing styles as props is not ideal. for instance here you are passing three low-level styles and they are the same every time. instead I would just include these values in the css if `dynamicWidth` is set to false:

`minWidth: ${p => p.dynamicWidth ? "none" : "295px"};`

that way these stylistic properties can be a subset of the *state* of `dynamicWidth=false`. when we pass lots of css values as properties it creates nearly infinite states and makes maintenance very hard.

Reply:
it also makes it very hard to reason about *why* stylistic differences occur because the changes have no name or logic.

Reply:
That makes sense, except... wasn't the whole point of this originally to be able to set the size of the hovercard? The dynamic-ness, though it's totally taken over, was really an off-shoot of that...